byway/src/main/java/com/google/sps/servlets/RouteServlet.java

```diff
@@ -29,8 +29,8 @@
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that handles data for stops */
```

Comment:
nit: change the comment to say handles data for the route

Reply:
done!